plugin-server/tests/main/process-event.test.ts

```diff
@@ -1459,7 +1459,7 @@ test('identify set', async () => {
     const [person] = await hub.db.fetchPersons()
     expect(await hub.db.fetchDistinctIdValues(person)).toEqual(['distinct_id1'])
     expect(person.properties).toEqual({ a_prop: 'test-1', c_prop: 'test-1' })
-    expect(person.is_identified).toEqual(false)
+    expect(person.is_identified).toEqual(true)
```

Comment:
What's the reason this is different now?

Reply:
From description:

> Also when we marked user as is_identified was complex to understand. Now it's simple - regardless if we merge or not if you pass a distinct_id to $identify, $create_alias or $merge_dangerously that distinct_id user will be marked as is_identified (alias and anon_id won't if merge won't happen)

Previously we marked user as is_identified after some checks (different IDs provided + not bad id like 'null' used), but before some other checks (e.g. trying to merge an already identified user) ... tldr it was complicated to explain what the actual condition is.

Reply:
relevant metabase query https://metabase.posthog.net/question/756-identify-without-anon-distinct-id - we'd always mark that person as is_identified now

Reply:
put differently:
1. if you call identify with anon-id = 'null' or similar we don't mark distinct_id as is_identified=true, after we do. same for alias and merge_dangerously (tied to alias being illegal distinct_id).
2. if you call identify without anon-id we previously didn't mark as is_identified=true, now we do (same for alias, but no-one uses alias without setting alias_id, so not relevant really)

Reply:
ok and the main thing to be careful off for is_identified is that we block merges in cases that it's already is_identified right? And the thing to consider there is the metabase query you link? The event mentioned there would not longer be able to be merged with other people, although it might be that that is never tried anyway?

Reply:
> ok and the main thing to be careful off for is_identified is that we block merges in cases that it's already is_identified right? 

👍 
> And the thing to consider there is the metabase query you link? 

 👍 

> The event mentioned there would not longer be able to be merged with other people, although it might be that that is never tried anyway?

Not exactly ... the real query for impact is complicated ... especially around what would be desired user behaviour. The Metabase query I linked shows user that would now be marked as `is_identified` and previously we would not mark that `true`. Most likely all those users were already `is_identified=true` or I suspect we'd want them to be marked that way.

A specific flow that would fail:
posthog-python:
```python
posthog.identify('session-id-for-A', {name: 'user A'})  // now we'll mark `session-id-for-A' as is_identfied
posthog.alias('user-A', 'session-id-for-A') // so merge will be blocked here, but previously it wasn't
```